pybr2021/cogs/schedules.py

```diff
@@ -0,0 +1,116 @@
+import asyncio
+import datetime
+from datetime import datetime, timedelta
+from pprint import pprint
+
+import bot_msg
+import httpx
+from decouple import config
+from discord.ext import commands, tasks
+from loguru import logger
+from pytz import timezone
+
+CALENDAR_URL='https://www.googleapis.com/calendar/v3/calendars/7siodq5un9gqbqd4mmgf2poiqs@group.calendar.google.com/events?key=AIzaSyAIn8DyZFtthupLozgwIX3NUURFMWEIPb4&timeMin=2021-10-11T00:00:00.000Z&timeMax=2021-10-18T00:00:00.000Z&singleEvents=true&maxResults=9999&timeZone=UTC'
+CALENDER_TIMEZONE= 'UTC'
+SHOW_TIMEZONE='America/Sao_Paulo'
+DISCORD_MSG_CHANNEL_ID='859819206584959007'  # Python Brasil 2021 > Geral
+DATE_FMT = "%d/%m/%Y %H:%M:%S"
+HOUR_FMT = "%H:%M"
+class Schedules(commands.Cog):
+
+    def  __init__(self,bot):
+        self.loop = asyncio.get_event_loop()
+        self._events = None
+        self.index = {}
+        self._bot = bot
+        self.alerts_type=["talk","closing","keynote","panel"]
+        self.load_events.start()
+
+    @tasks.loop(minutes=1)
```

Comment:
```suggestion
    @tasks.loop(minutes=60)
```
Os devem mudar muito pouco para gente fazer uma query por minuto. Melhor diminuir a concorren